cmd/utils.go

```diff
@@ -36,6 +36,14 @@ import (
 	"github.com/pkg/profile"
 )
 
+// Close Http tracing file.
+func stopHTTPTrace() {
+	if globalHTTPTraceFile != nil {
+		errorIf(globalHTTPTraceFile.Close(), "Error closing file")
```

Comment:
`Unable to close httpTraceFile %s", globalHTTPTraceFile.Name())`

Reply:
done